### ISSUE TITLE
Error : Typed property League\CommonMark\Parser\InlineParserEngine::$parsers must not be accessed before initialization

### ISSUE THREAD

**Issue by lezhnev74** [thephpleague/commonmark#908]
### Version(s) affected

2.3.4 (under php 8.1.8)

### Description

Unexpected error when parsing \```\n```

### How to reproduce

```php
$env = new Environment();
$p = new MarkdownParser($env);
$p->parse('```\n```');
```

**Comment by lezhnev74** [comment]
Should I provide a failing test and a patch to fix this?

**Comment by colinodell** [comment]
Thanks for the report! This looks easy enough to fix - I'll prepare and release the patch shortly :)

**Comment by colinodell** [comment]
Fixed in 2.3.5!
